plugins/source/gcp/resources/services/serviceusage/services_fetch.go

```diff
@@ -0,0 +1,44 @@
+package serviceusage
+
+import (
+	"context"
+
+	serviceusage "cloud.google.com/go/serviceusage/apiv1"
+	pb "cloud.google.com/go/serviceusage/apiv1/serviceusagepb"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugins/source/gcp/client"
+	"google.golang.org/api/iterator"
+)
+
+func fetchServices(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	if len(c.EnabledServices) > 0 {
+		for _, svc := range c.EnabledServices[c.ProjectId] {
+			res <- svc.(*pb.Service)
```

Comment:
Isn't  `svc` a `map[string]interface{}`?

Reply:
`svc` is just `interface{}` the skipped key value (`_`) is a `string`